server/zanata-war/src/test/java/org/zanata/adapter/GettextAdapterTest.java

```diff
@@ -105,5 +111,48 @@ public void testGettextReference() {
         assertThat(potEntryHeader.getReferences().get(0)).isEqualTo("reference");
     }
 
+    @Test
+    public void testTranslatedGettext() {
+
+        TranslationsResource transResource = new TranslationsResource();
+        addTranslation(transResource, "0293301ed6a54b7e4503e74bba17bf11", "Carpeta padre", ContentState.Approved);
+        addTranslation(transResource, "47a0be8d1015d526a1fbaa56c3102135", "Asunto:", ContentState.Translated);
+        addTranslation(transResource, "49ab28040dfa07f53544970c6d147e1e", "Conectar", ContentState.NeedReview);
```

Comment:
I agree, and have discussed this with @spathare21 as a near-future update.  It's my fault they exist.
This is what I came up with:
https://github.com/zanata/zanata-platform/pull/192/commits/a48a088f5abb8bf0055ecdd55e897a4d775e2724